src/app/routes/getInitialData/utils/getRouteProps/index.js

```diff
@@ -12,11 +13,15 @@ const getRouteProps = (routes, url) => {
 
   const amp = path(['amp'], params);
   const service = path(['service'], params);
+  const rawVariant = path(['variant'], params);
```

Comment:
Renaming as variantPath would be useful for readability since this gives  `/cyr` `/lat` `/simp` `/trad` etc. Had to log locally for this. Also adding a comment here would help.

Reply:
Done :)